components/edit-collective/sections/Tiers.js

```diff
@@ -387,38 +394,65 @@ class Tiers extends React.Component {
 
   render() {
     const { intl, defaultType = 'TICKET' } = this.props;
+    let defaultIsChecked = false;
+    if (this.props.collective.settings?.disableCustomContributions === undefined) {
+      defaultIsChecked = false;
+    } else {
+      defaultIsChecked = this.props.collective.settings?.disableCustomContributions;
+    }
 
     return (
-      <div className="EditTiers">
-        <style jsx>
-          {`
-            :global(.tierActions) {
-              text-align: right;
-              font-size: 1.3rem;
-            }
-            :global(.field) {
-              margin: 1rem;
-            }
-            .editTiersActions {
-              text-align: right;
-              margin-top: -10px;
-            }
-            :global(.tier) {
-              margin: 3rem 0;
-            }
-          `}
-        </style>
-
-        <div className="tiers">
-          <h2>{this.props.title}</h2>
-          {this.state.tiers.map(this.renderTier)}
-        </div>
-        <div className="editTiersActions">
-          <Button className="addTier" bsStyle="primary" onClick={() => this.addTier({})}>
-            {intl.formatMessage(this.messages[`${defaultType}.add`])}
-          </Button>
-        </div>
-      </div>
+      <Mutation mutation={updateSettingsMutation}>
+        {(editSettings, { loading }) => (
+          <div className="EditTiers">
+            <style jsx>
+              {`
+                :global(.tierActions) {
+                  text-align: right;
+                  font-size: 1.3rem;
+                }
+                :global(.field) {
+                  margin: 1rem;
+                }
+                .editTiersActions {
+                  text-align: right;
+                  margin-top: -10px;
+                }
+                :global(.tier) {
+                  margin: 3rem 0;
+                }
+              `}
+            </style>
+            <div className="tiers">
+              <h2>{this.props.title}</h2>
+              <Flex flexDirection="column" alignItems="center" justifyContent="center" minWidth={300}>
+                <StyledCheckbox
+                  name="custom-contributions"
+                  label={intl.formatMessage(this.messages['customContributions.label'])}
+                  defaultChecked={!defaultIsChecked}
```

Comment:
This is confusing. Why not `defaultChecked={defaultIsChecked}`?

Reply:
> When saving with an unchecked state, collective.settings.disableCustomContributions will be set to true.

When saving with unchecked state it should be true that means intial value is false and we have to negate it to make `defaultChecked=true` ( Deafult state )

Reply:
The variable names could be clearer, sounds like `defaultIsChecked` should be more like `hasCustomContributionsDisabled`